test/e2e/utils/cleanup/cleanup.go

```diff
@@ -0,0 +1,43 @@
+package cleanup
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/1Password/terraform-provider-onepassword/v2/internal/onepassword/model"
+	"github.com/1Password/terraform-provider-onepassword/v2/test/e2e/utils/client"
+)
+
+// RegisterItemCleanup registers a cleanup function that will delete an item even if the test fails.
+func RegisterItemCleanup(t *testing.T, itemUUID, vaultID string) {
+	if itemUUID == "" {
+		return
```

Comment:
Do we expect `itemUUID` will be always passed? Maybe return an error in case if not?